feat: support extend rules (#90)
joshbolduc/vscode-commitlint: README.md

- Runs commitlint against your commit message as you write it
- Reports lint [errors and warnings in the editor](https://code.visualstudio.com/docs/editor/editingevolved#_errors-warnings)
- Highlights relevant parts of the commit based on the specific issue
- Auto detects commitlint configuration by default
- Supports [all commitlint rules](https://commitlint.js.org/#/reference-rules?id=available-rules)

## Settings

> ### `commitlint.config.file`
> 
> Path to a custom commitlint configuration file. Relative paths are resolved based on the workspace root. Leave blank to auto-detect.
> 
> ### `commitlint.config.extend.rules`
> 
> Commitlint rules which will be extended.
> 
> **Example configuration in JSON type:**

Reviewer: ```suggestion
### `commitlint.config.file`

Path to a custom commitlint configuration file. Relative paths are resolved based on the workspace root. Leave blank to auto-detect.

### `commitlint.config.extend.rules`

Commitlint rules which will be extended.

**Example configuration as JSON syntax:**
```
